mamba_gator/handlers.py

```diff
@@ -23,6 +23,7 @@
 from .log import get_logger
 from jupyter_server.base.handlers import APIHandler
 from jupyter_server.utils import url_path_join
+from conda.base.context import context
```

Comment:
Is there an alternative option to get the subdir information - like using a command line?

I'm worried that that import will bring trouble for users installing gator in a non-root environment in which conda python package is not available.

Reply:
I couldn't find a way to do this via the command line, other than parsing the output of `conda list --explicit`, which might be worth it now I know the concerns of importing conda.